.changeset/green-spies-arrive.md

```diff
@@ -0,0 +1,24 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+
+Allow `Record` typed object to be used to construct MapNodes.
+
+It is now allowed to construct MapNodes from `Record` typed objects, similar to how maps are expressed in JSON.
+
+Before this change, an `Iterable<string, Child>` was required, but now an object like `{key1: Child1, key2: Child2}` is allowed.
+
+Full example using this new API:
+```typescript
+class Schema extends schemaFactory.map("ExampleMap", schemaFactory.number) {}
+const fromRecord = new Schema({ x: 5 });
+```
+
+This new feature makes it possible for schema,
```

Comment:
Rewording suggestion. I think I got the meaning right.
```suggestion
This new feature makes it possible for schemas to construct a tree entirely from JSON compatible objects using their constructors, as long as they do not require unhydrated nodes to differentiate ambiguous unions.
```

Reply:
If you want to suggest changing multiple lines, you should select multiple lines when making the comment. Applying that suggestion would just replace line 18 with that, which I'm pretty sure is not what you intended.

Reply:
applied slightly reworked version of change separate from this thread. Good rephrasing!